app/sqs.go

```diff
@@ -3,6 +3,8 @@ package app
 import (
 	"sync"
 	"time"
+
+	"github.com/p4tin/goaws/app/common"
```

Comment:
The base app package should never import a dependancy from this project as it is in place to avoid circular dependacies - also no code goes in any of the 'app' package files - they are strictly for struct and interface declarations.

Reply:
@p4tin - I moved the methods code to `app/gosqs/gosqs.go`, but you should consider adding methods to the structs in `app`. 

The pattern you're following is missing the point of having structs you can attach methods to.